docs/gettingstarted/structuring.md

```diff
@@ -30,7 +30,8 @@ All components of @Mod() are written inside the parantheses and chained together
 Components:
 
 `modid` [Required]
-This identifies your mod uniquely to Forge and should probably be the first thing you enter. It's only visible internally.
+This identifies your mod uniquely to Forge and should probably be the first thing you enter. It's only used internally, but can be seen by users who dig around. Other mods also require this ID if they want to find each other. (APIs for example.)
+If you want to know more about annotations in general you can check out [Oracle](https://docs.oracle.com/javase/tutorial/java/annotations/index.html)'s docs on it.
```

Comment:
You are linking to the docs, not to Oracle. Hence the link should be on either "docs" or "Oracle's docs" as a whole.